site/docs/v1/tech/apis/identity-providers/nintendo.adoc

```diff
@@ -0,0 +1,46 @@
+---
+layout: doc
+title: Nintendo Identity Provider APIs
+description: APIs for creating, retrieving, updating and deleting Nintendo identity providers
+---
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+== Overview
+
+[NOTE.since]
+====
+This API has been available since 1.36.0
+====
+
+The Nintendo identity provider type will use the Nintendo OAuth login API. It will also provide a `Login with Nintendo` button on FusionAuth's login page.
+
+FusionAuth will leverage the `/users/me` API that is part of the Nintendo specification. The email address and username returned in the response will be used to create or lookup the existing User. Additional claims from the response can be used to reconcile the User in FusionAuth by using an Nintendo Reconcile Lambda. Unless you assign a reconcile lambda to this provider or configure the IdP options to use different claims, the `email` and `preferred_username` will be used from the available claims returned by the Nintendo identity provider.
```

Comment:
> The email address and username returned in the response will be used to create or lookup the existing User.

This is true if it is returned, and based upon the linking strategy.

Reply:
should be fixed